test/start/file/test.test.js

```diff
@@ -19,5 +19,7 @@ const tape = require('tape');
 
 tape('from-file serviceName test', function (t) {
   t.equals(agent._conf.serviceName, 'from-file');
+  t.equals(agent._conf.active, false);
```

Comment:
nitpick: we are doing more than the test title states so I'd suggest either to change the tile and make use of the `msg` param of `t.equals` assert function.

Something like
```js
tape('from-file configuration test', function (t) {
  t.equals(agent._conf.serviceName, 'from-file', 'serviceName comes from config file');
  t.equals(agent._conf.active, false, 'false values from config file override defaults');
  t.equals(agent._conf.captureBody, 'off', 'existing defaults are preserved');
  t.end();
});
```

Reply:
Updated